packages/client-sdk-nodejs/src/config/middleware/impl/experimental-metrics-middleware.ts

```diff
@@ -24,50 +24,52 @@ const FIELD_NAMES: Array<string> = [
 ];
 
 export interface ExperimentalRequestMetrics {
-  /**
-   * number of requests active at the start of the request
-   */
-  numActiveRequestsAtStart: number;
-  /**
-   * number of requests active at the finish of the request (including the request itself)
-   */
-  numActiveRequestsAtFinish: number;
-  /**
-   * The generated grpc object type of the request
-   */
-  requestType: string;
-  /**
-   * The grpc status code of the response
-   */
-  status: number;
-  /**
-   * The time the request started (millis since epoch)
-   */
-  startTime: number;
-  /**
-   * The time the body of the request was available to the grpc library (millis since epoch)
-   */
-  requestBodyTime: number;
-  /**
-   * The time the request completed (millis since epoch)
-   */
-  endTime: number;
-  /**
-   * The duration of the request (in millis)
-   */
-  duration: number;
-  /**
-   * The size of the request body in bytes
-   */
-  requestSize: number;
-  /**
-   * The size of the response body in bytes
-   */
-  responseSize: number;
-  /**
-   * The ID of the specific connection that made the request
-   */
-  connectionID: string;
+  MomentoMetricsMiddleware: {
```

Comment:
not a strong opinion but I think `momento. numActiveRequestsAtStart` or `momento.duration` will be a cleaner UX than `MomentoMetricsMiddleware.duration`

Reply:
I like that better too, updated